src/main/java/org/broadinstitute/hellbender/tools/spark/sv/AlignmentRegion.java

```diff
@@ -69,6 +76,19 @@ public AlignmentRegion(final GATKRead read) {
         }
     }
 
+    public AlignmentRegion(final Kryo kryo, final Input input) {
+        this.contigId = input.readString();
+        this.assemblyId = input.readString();
+        this.forwardStrandCigar = TextCigarCodec.decode(input.readString());
```

Comment:
If you haven't already, you should check that `Cigar.toString()` and `TextCigarCodec.decode(String)` are precise inverses in all cases.  Maybe there's a unit test that already does this.  I haven't looked.